app/controllers/spam2_controller.rb

```diff
@@ -3,7 +3,7 @@ class Spam2Controller < ApplicationController
 
   def _spam
     if logged_in_as(%w(moderator admin))
-      @nodes = Node.order('nid DESC')
+      @nodes = Node.order('nid DESC').paginate(page: params[:page])
```

Comment:
Oops, sorry -- looks like you missed one here -- can you fix this too? Thanks!

Reply:
@jywarren  Sure, Thanks!!